.editorconfig

```diff
@@ -0,0 +1,153 @@
+﻿###############################
+# Core EditorConfig Options   #
+###############################
```

Comment:
I believe this file was mirrored from somewhere? If it is, let us add an URL to original copy in the comments, so further generations of developers could continue to update it from the source?

Reply:
Source added